comms/WorkspaceItem.cpp

```diff
@@ -332,6 +332,11 @@ class CWorkspaceItem : public IWorkspaceItem, public clib::CLockableUnknown
         return m_attribute;
     }
 
+    IAttribute * GetAttributePointer() const
+    {
+        return m_attribute;
+    }
+
```

Comment:
Minor:  Since returning an IAttibute * is potentially "thread" dangerous (judging by the code above it), it might be better to make this call:  `matches(otherWorkspaceItem)` and do the compare internally?